.gitignore

```diff
@@ -0,0 +1,159 @@
+.idea/
+
```

Comment:
I prefer to add a line of the form

```
## From https://github.com/github/gitignore/blob/main/Python.gitignore
```

here so this is attributed to the gitignore repository correctly.